src/sensors/mpu9250sensor.cpp

```diff
@@ -166,16 +182,29 @@ void MPU9250Sensor::motionLoop() {
 
     quaternion = correction * quat;
 #else
-    unsigned long now = micros();
-    unsigned long deltat = now - last; //seconds since last update
-    last = now;
-    getMPUScaled();
+
+    union fifo_sample_raw buf;
+    uint16_t remaining_samples;
+    // TODO: would it be faster to read multiple samples at once
+    while (getNextSample (&buf, &remaining_samples)) {
```

Comment:
I started with a relatively complicated design that involved the ability to read multiple samples into a buffer at once, and then maybe focused too much on avoiding issues with strict aliasing, but settled on this simpler design in the end.

In essence, this TODO is asking the question "how much time is wasted doing individual i2c transactions instead of trying to batch them"? If the time is neglegible, then it's not important. If it's a lot, then maybe we should try adding this in.

might want to write a microbenchmark to figure this out, if we care enough to do so.